static/src/javascripts/projects/commercial/modules/comment-adverts.ts

```diff
@@ -1,6 +1,4 @@
-import type { SizeKeys } from '@guardian/commercial-core';
-import { adSizes, createAdSize, createAdSlot } from '@guardian/commercial-core';
-import { toGoogleTagSize } from 'common/modules/commercial/lib/googletag-ad-size';
+import { adSizes, createAdSlot } from '@guardian/commercial-core';
```

Comment:
Not sure if this module is hanging around (as it doesn't currently work), but have updated it to no longer use data-attributes to upgrade the slot and use a new method on the `Advert` class `updateSizeMapping`.

Reply:
I think we kept this as we want to apply this logic to DCR but just haven't got to the ticket.

Reply:
sweet, yeah I had an inkling!